### ISSUE TITLE
improve content accessibility by making [here] links more descriptive

### ISSUE THREAD

**Issue by serahkiburu** [carpentries/carpentries.org-archive#990]
As @zkamvar rightly pointed out in PR #867 a few months ago, having a link point to "here" is not accessible for users who use screen-readers to scan for links: https://webaccess.berkeley.edu/ask-pecan/click-here

We invite you to help us make our content more accessible by [creating Pull Requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) in this repository with suggestions for more descriptive text to replace [here] links across our resource. 

(not required but nice to have)Where possible, you can also [mention this issue in your Pull Request](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) to enable us to keep track of accessibility improvements made on this specific issue.

Thank you for your contributions.

**Comment by maneesha** [comment]
Addressed with https://github.com/carpentries/carpentries.org/pull/1147
